app/retail/views.py

```diff
@@ -1654,51 +1613,120 @@ def tribes(request):
+        }
+    ]
+
+    press = [
+        {
+            'link': 'https://twit.tv/shows/floss-weekly/episodes/474',
+            'img': 'v2/images/press/floss_weekly.jpg'
+        },
+        {
+            'link': 'https://epicenter.tv/episode/257/',
+            'img': 'v2/images/press/epicenter.jpg'
+        },
+        {
+            'link': 'http://www.ibtimes.com/how-web-30-will-protect-our-online-identity-2667000',
+            'img': 'v2/images/press/ibtimes.jpg'
+        },
+        {
+            'link': 'https://www.forbes.com/sites/jeffersonnunn/2019/01/21/bitcoin-autonomous-employment-workers-wanted/',
+            'img': 'v2/images/press/forbes.jpg'
+        },
+        {
+            'link': 'https://unhashed.com/cryptocurrency-news/gitcoin-introduces-collectible-kudos-rewards/',
+            'img': 'v2/images/press/unhashed.jpg'
+        },
+        {
+            'link': 'https://www.coindesk.com/meet-dapp-market-twist-open-source-winning-developers/',
+            'img': 'v2/images/press/coindesk.png'
+        },
+        {
+            'link': 'https://softwareengineeringdaily.com/2018/04/03/gitcoin-open-source-bounties-with-kevin-owocki/',
+            'img': 'v2/images/press/se_daily.png'
+        },
+        {
+            'link': 'https://www.ethnews.com/gitcoin-offers-bounties-for-ens-integration-into-dapps',
+            'img': 'v2/images/press/ethnews.jpg'
+        },
+        {
+            'link': 'https://www.hostingadvice.com/blog/grow-open-source-projects-with-gitcoin/',
+            'img': 'v2/images/press/hosting-advice.png'
         }
     ]
 
+    articles = [
+        {
+            'link': 'https://medium.com/gitcoin/progressive-elaboration-of-scope-on-gitcoin-3167742312b0',
+            'img': static("v2/images/medium/1.png"),
+            'title': _('Progressive Elaboration of Scope on Gitcoin'),
+            'description': _('What is it? Why does it matter? How can you deal with it on Gitcoin?'),
+            'alt': 'gitcoin scope'
+        },
+        {
+            'link': 'https://gitcoin.co/blog/commit-reveal-scheme-on-ethereum/',
+            'img': static("v2/images/medium/2.png"),
+            'title': _('Commit Reveal Scheme on Ethereum'),
+            'description': _('Hiding Actions and Generating Random Numbers'),
+            'alt': 'commit reveal scheme'
+        },
+        {
+            'link': 'https://medium.com/gitcoin/announcing-open-kudos-e437450f7802',
+            'img': static("v2/images/medium/3.png"),
+            'title': _('Announcing Open Kudos'),
+            'description': _('Our vision for integrating Kudos in any (d)App'),
+            'alt': 'open kudos'
+        }
+    ]
     context = {
-        'plans': plans,
         'tribes': tribes,
         'testimonials': testimonials,
-        'reasons': reasons
+        'reasons': reasons,
+        'articles': articles,
+        'press': press,
+        'top_tribes': top_tribes
```

Comment:
tribes already shows the top tribes ! Why do we have a duplicate one ?

![image](https://user-images.githubusercontent.com/5358146/79690352-a342bb00-8277-11ea-924b-6c9f7ca33de9.png)